src/index.d.ts

```diff
@@ -16,6 +16,9 @@ import {
   TransporterOptions
 } from 'electron-notarize/lib/types';
 import { SignOptions } from 'electron-osx-sign';
+import type { makeUniversalApp } from '@electron/universal';
+
+type MakeUniversalOpts = Parameters<typeof makeUniversalApp>[0]
```

Comment:
note: this leverages `Parameters` in TS to get the args for `makeUniversalApp` since `@electron/universal` doesn't expose them.